<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OmgTaxaPageBase</class>
 <widget class="QWizardPage" name="OmgTaxaPageBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WizardPage</string>
  </property>
  <property name="title">
   <string>Selection of taxa</string>
  </property>
  <property name="subTitle">
   <string>Select one or more taxa to use in creation of your models.</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="OmgListWidget" name="omglistTaxa" native="true">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
     <property name="focusPolicy">
      <enum>Qt::TabFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>OmgListWidget</class>
   <extends>QWidget</extends>
   <header>omglistwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
